### PR TITLE
Add w as default bind for dummy hammer

### PR DESCRIPTION
**Pull request by def-**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Comment by heinrich5991** [comment]
People use w for jump I think. Was there a motivation for choosing w?

**Comment by def-** [comment]
It's the bind I've been using for it for years. I think my idea was that `w` in the `wasd` is for going up, and hammerfly goes up.

**Comment by def-** [comment]
If you can suggest another key, I can use that

**Comment by heinrich5991** [comment]
image

I'd guess H?

**Comment by def-** [comment]
that's already used for happy emote :D

**Comment by heinrich5991** [comment]
Would adding it as H anyway work, instead of the happy emote? Or does that not fix the problem because existing players will still have the happy emote?

**Comment by def-** [comment]
That's ok, overwrote the happy emote and removed the other emote binds for consistency

File: src/game/client/components/binds.cpp [hunk 1]
```diff
@@ -466,11 +466,13 @@ void CBinds::SetDDRaceBinds(bool FreeOnly)
 		Bind(KEY_C, "say /rank", FreeOnly);
 		Bind(KEY_V, "say /info", FreeOnly);
 		Bind(KEY_B, "say /top5", FreeOnly);
-		Bind(KEY_X, "emote 14", FreeOnly);
-		Bind(KEY_H, "emote 2", FreeOnly);
-		Bind(KEY_M, "emote 5", FreeOnly);
 		Bind(KEY_S, "+showhookcoll", FreeOnly);
 		Bind(KEY_X, "toggle cl_dummy 0 1", FreeOnly);
+		Bind(KEY_W, "toggle cl_dummy_hammer 0 1", FreeOnly);
```
Comment (heinrich5991) [inline src/game/client/components/binds.cpp:471]: Did you intentionally keep the duplicate bind?

Comment (def-) [reply src/game/client/components/binds.cpp:471]: Which duplicate?

Comment (def-) [reply src/game/client/components/binds.cpp:471]: Oops, will remove

**Comment by heinrich5991** [comment]
bors r+

Thanks.

**Pull request merged**